src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -41,27 +42,29 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the person identified "
             + "by the index number used in the displayed person list. "
             + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
+            + "Parameters: INDEX (must be a positive integer in the displayed list) "
             + "[" + PREFIX_NAME + "NAME] "
             + "[" + PREFIX_PHONE + "PHONE] "
             + "[" + PREFIX_EMAIL + "EMAIL] "
             + "[" + PREFIX_ADDRESS + "ADDRESS] "
             + "[" + PREFIX_CALLED + "CALLED]"
             + "[" + PREFIX_GENDER + "GENDER] "
             + "[" + PREFIX_AGE + "AGE] "
-            + "[" + PREFIX_INTEREST + "(optional index) INTEREST]...\n"
+            + "[" + PREFIX_INTEREST + "INTEREST_TO_BE_ADDED]"
+            + "[" + PREFIX_INTEREST + "(INDEX) remove]"
+            + "[" + PREFIX_INTEREST + "(OPTIONAL INDEX) INTEREST]...\n"
```

Comment:
change "..." to the ones we use in the UG
remove "optional" from "optional index"